example/arq_prometheus_worker/scripts/init_redis

```diff
@@ -1,18 +1,20 @@
 #!/bin/sh -e
 
-# if a redis container is running, print instructions to kill it and exit
+# Check if a Redis container is already running
 RUNNING_CONTAINER=$(docker ps --filter 'name=redis' --format '{{.ID}}')
 if [ -n "$RUNNING_CONTAINER" ]; then
-  echo >&2 "there is a redis container already running, kill it with"
-  echo >&2 "    docker kill ${RUNNING_CONTAINER}"
+  echo >&2 "A Redis container is already running. Please stop it before running this script."
```

Comment:
Wouldn't it be better to keep the command for the end user to know what needs to be done in order to stop the container?